tests/EdgeToEdge/RouteNotFoundTest.php

```diff
@@ -29,7 +29,7 @@ public function testGivenUnknownRouteAndJsonRequest_responseIsJSON(): void {
 		$client = $this->createClient();
 		$client->request( 'GET', '/kittens', [], [], [ 'HTTP_ACCEPT' => 'application/json' ] );
 
-		$this->assertJsonResponse( [ 'ERR' => 'No route found for "GET /kittens"' ], $client->getResponse() );
```

Comment:
Please create a separate commit that fixes this, so we won't be confused  why this otherwise unrelated piece of code had to change.

Reply:
Whoops, I amended by reflex